api/app/Mail/EventCreatedSlackToOwner.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Mail;
+
+use App\Entity\Event;
+use App\Entity\EventType;
+use App\Entity\User;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Notifications\Messages\SlackMessage;
+use Illuminate\Queue\SerializesModels;
+
+class EventCreatedSlackToOwner extends SlackMessage implements ShouldQueue
```

Comment:
It should be done properly. You do not need `getSlackMessage` method because the class is `SlackMessage` by itself. The class should be named as `EvantCreatedSlackMessage` and put to `app/Notifications/SlackMessages` folder.

Reply:
How do you want it to be?

Reply:
Did you check `Illuminate\Notifications\Messages\SlackMessage` class sources? It is just a DTO. You can extend it and add the constructor that will receive the event and set the needed parameters, or you can create a static method that will do the same.